setup_git.md

```diff
@@ -1,4 +1,7 @@
 ### SSH connect local to github
 * https://gist.github.com/john-auld/a55ffe44e9965623f530cb487e33cebd
 * When add key: ssh-add testkey instead of adding the path
-* push origin master not working
\ No newline at end of file
+* push origin master not working
+
+### Branching 
```

Comment:
I just made a comment on this line, and it is for communication between owner of this branch and teammate